incubator/java-openliberty/templates/kafka/src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -1,10 +1,8 @@
-mp.messaging.connector.liberty-kafka.bootstrap.servers=kafka:9092
+mp.messaging.connector.liberty-kafka.bootstrap.servers=localhost:9093
```

Comment:
Could you explain why the port changes too?

Reply:
The Kafka has two separate listeners configured for inside the Docker network and outside, so they must listen on two separate ports.  Two listeners are required because we need to advertise different hostnames. Not sure if we need to explain this in the README?

Reply:
Ahh that makes sense. I think previously I'd seen Kafka using 9092 for plaintext and 9093 for TLS...